### DISCUSSION TITLE
Can't make a new account

### DISCUSSION THREAD

**Discussion by RocioKanmi** [stoatchat/discussions#769] [Q&A] [answered]
Hi there, im not sure where i can post this, is not a bug or something ig.
Just we can't make a new account to start to see this aplication.

When i try to make the account i got "Internalerror", same my friend try to register yesterday and nothing. We are trying to make the new account and nothing yet.

**Comment by IAmTomahawkx** [comment]
Hi, this is indeed an error on our end. We're working on it. Thanks!

**Reply by insertish** [reply, marked as answer]
(just following up to say it has been fixed)

**Reply by Chopa4ok** [reply]
how u fixed it? I got this problem and cant solve it

**Reply by touchfalls** [reply]
the server can't handle it, sometimes it turns on the registration service so that it doesn't shut down automatically
